src/platform-integration/ios/platform-views.md

```diff
@@ -242,6 +246,10 @@ and the platform view provides a reference to the
                              binaryMessenger:_messenger];
 }
 
+- (NSObject<FlutterMessageCodec>*)createArgsCodec {
```

Comment:
This protocol method is optional. Implementing is necessary when `createWithFrame`'s `arguments` parameter is not nil. See: https://github.com/flutter/engine/blob/main/shell/platform/darwin/ios/framework/Headers/FlutterPlatformViews.h#L45-L49

We should add a comment here to explain it to avoid making people believe this method is required. How about:
// Implementing this method is only necessary when the `arguments` in `createWithFrame` is not nil.

Reply:
Dear @cyanglaz, I added comment as your request.